### PR TITLE
sql: Add EXPLAIN ANALYZE documentation / example

### LINKED ISSUE #690: sql: Document EXPLAIN ANALYZE
EXPLAIN ANALYZE is great! 

We should amend the example here to show an example with analyze:
[./sql/understanding-the-query-execution-plan.md#example-usage](https://github.com/pingcap/docs/blob/master/sql/understanding-the-query-execution-plan.md#example-usage)

### PR DESCRIPTION
**Pull request by morgo**
Fixes #690

### CODE REVIEW

**Comment by morgo** [comment]
PTAL @lysu

File: sql/understanding-the-query-execution-plan.md [hunk 1]
```diff
@@ -68,6 +68,31 @@ mysql> EXPLAIN SELECT count(*) FROM trips WHERE start_date BETWEEN '2017-07-01 0
 
 In the revisited `EXPLAIN` you can see the count of rows scanned has reduced via the use of an index. On a reference system, the query execution time reduced from 50.41 seconds to 0.00 seconds!
 
+## <span id="explain-analyze-output-format">`EXPLAIN ANALYZE` output format</span>
+
+As an extention to `EXPLAIN`, `EXPLAIN ANALYZE` will execute the query and provide additional execution statistics in the `operator_info` column is as follows:
```
Comment (shenli) [inline sql/understanding-the-query-execution-plan.md]: s/extention/extension/

Comment (shenli) [reply sql/understanding-the-query-execution-plan.md]: Should we remove the 'is' before 'as follows'?

**Comment by shenli** [comment]
@lysu PTAL

**Comment by shenli** [comment]
LGTM

**Comment by morgo** [comment]
@CaitinChen PTAL

**Comment by lysu** [comment]
LGTM

**Pull request merged**